### PR TITLE
perfscale-vshpere8

### PR DESCRIPTION
**Pull request by qiliRedHat**
https://issues.redhat.com/browse/OCPQE-14084

### CODE REVIEW

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-vshpere8-4.14-ocp-qe-perfscale-ci-tests-net-perf

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-vshpere8-4.14-ocp-qe-perfscale-ci-tests-net-perf

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-vshpere8-4.14-ocp-qe-perfscale-ci-tests-net-perf

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-vshpere8-4.14-ocp-qe-perfscale-ci-tests-net-perf

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-vshpere8-4.14-ocp-qe-perfscale-ci-tests-net-perf

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__vshpere8-4.14.yaml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: ocp-qe-perfscale-ci-tests-kube-burner
+  cron: '@yearly'
```
Comment (paigerube14) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__vshpere8-4.14.yaml:26]: is yearly how much we want to run this test?

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-vshpere8-4.14-vsphere-8-ipi-ovn-etcd-encryption-p2-f14

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-vshpere8-4.14-ocp-qe-perfscale-ci-tests-net-perf

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-vshpere8-4.14-ocp-qe-perfscale-ci-tests-net-perf

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-vshpere8-4.14-vsphere-8-ipi-ovn-test

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-vshpere8-4.14-ocp-qe-perfscale-ci-tests-net-perf

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-vshpere8-4.14-ocp-qe-perfscale-ci-tests-net-perf

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-vshpere8-4.14-vsphere-8-ipi-ovn-test

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-vshpere8-4.14-ocp-qe-perfscale-ci-tests-kube-burner

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-vshpere8-4.14-ocp-qe-perfscale-ci-tests-net-perf

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-vshpere8-4.14-ocp-qe-perfscale-ci-tests-vsphere-8-network-perf

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-vshpere8-4.14-vsphere-8-ipi-ovn-test

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-vshpere8-4.14-ocp-qe-perfscale-ci-tests-vsphere-8-router-perf

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-vshpere8-4.14-ocp-qe-perfscale-ci-tests-vsphere-8-router-perf

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-vshpere8-4.14-ocp-qe-perfscale-ci-tests-vsphere-8-network-perf

**Comment by qiliRedHat** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-vshpere8-4.14-ocp-qe-perfscale-ci-tests-vsphere-8-router-perf

**Comment by qiliRedHat** [comment]
"In prow, vsphere include several different segment from different vpshere cluster, inlcuding poc-vsphere-8, zonal-vsphere, and so on, so any job running on vsphere cluster, would be randomly run in different vpshere cluster,  Presently we did not add the new ibmc segment into the cluster profile yet", and because vsphere-8 on IBMC is supported in Jenkins CI, I'll hold on moving scale-ci to prow for vshpere. Closing this PR.

**Pull request closed without merging**